src/library/dlgtrackinfo.cpp

```diff
@@ -744,7 +748,10 @@ mixxx::UpdateResult DlgTrackInfo::updateKeyText() {
 }
 
 void DlgTrackInfo::displayKeyText() {
-    const QString keyText = m_trackRecord.getMetadata().getTrackInfo().getKeyText();
+    const QString keyText =
+            KeyUtils::tryConvertKeyString(
+                    m_trackRecord.getMetadata().getTrackInfo().getKeyText(),
+                    m_notation);
```

Comment:
I think it is better to use the same string the track object will return for display purpose: 
```suggestion
    const QString keyText = KeyUtils::keyToString(m_record.getKeys().getGlobalKey());
```
this should work without `m_notation` which can be probably be removed, (not tested)